document/core/syntax/modules.rst

```diff
@@ -363,3 +363,9 @@ In each index space, the indices of imports go before the first index of any def
 
 .. note::
    In the current version of WebAssembly, only *immutable* globals may be imported.
+
+   Unlike export names, import names are not necessarily unique.
+   It is possible to import the same |IMODULE|/|INAME| pair multiple times.
+   Even if such imports have different types a module can still be instantiated by :ref:`embedders <embedder>` for which the type difference is irrelevant or that take the expected types into account when resolving imports.
+   However, embedders are not required to support such "overloading",
```

Comment:
For my reference, do the quotes here mean this word is being misused?

Reply:
Removed the quotes. It was more because it all depends on the host's interpretation. For example, with JavaScript it isn't really overloading, because the name denotes the same single object (and it fails in all cases where it doesn't). Similarly, another hypothetical embedder could interpret the names as a reference to a single generic definition, with the types as different instantiation parameters.